File: src/syntax/lesson14_scanner/ScannerDemo.java
package syntax.lesson14_scanner;

import java.util.Scanner;

public class ScannerDemo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter your name: ");
        var name = scanner.nextLine().trim();

        System.out.println("Enter your age: ");
        int age = scanner.nextInt();

        System.out.println("Hello, my name is " + name + "! I am " + age + " years old.");

        scanner.close();
    }
}

File: src/syntax/lesson14_scanner/ScannerDemo.kt
package syntax.lesson14_scanner

import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)
    print("Enter your name: ")
    val name = scanner.nextLine()
    print("Enter your age: ")
    val age = scanner.nextInt()

    println("Hello, my name is $name! I'm a $age years old.")
    scanner.close()
}